composer.json

```diff
@@ -23,5 +26,10 @@
         "branch-alias": {
             "dev-master": "1.1-dev"
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
```

Comment:
Just noticed this. Is this actually required for anything? I haven't come across this before and we don't use in [mongo-php-library](https://github.com/mongodb/mongo-php-library).

Looking at `phpcs.xml.dist`, I only see references to the [standard rules that ship with phpcs](https://github.com/squizlabs/PHP_CodeSniffer/tree/master/src/Standards) and Slevomat, which we explicitly require above.